tests/DCPS/XTypes/Pub/XTypesPublisher.cpp

```diff
@@ -182,11 +179,26 @@ int ACE_TMAIN(int argc, ACE_TCHAR* argv[])
     }
   }
 
+  if (TheServiceParticipant->get_security()) {
+#if !defined(OPENDDS_SECURITY)
+    ACE_DEBUG((LM_DEBUG, "Security requested, but not enabled\n"));
+    return 0;
+#endif
+  }
+
```

Comment:
```suggestion
```

`TheServiceParticipant->get_security()` isn't there if `OPENDDS_SECURITY` isn't defined.

Reply:
How would I detect a situation when security is enabled in the ini file, but OpenDDS was built without security?

Reply:
Only run with that ini file in `tests/security/security_tests.lst`.